riff-cli/pkg/osutils/osutils.go

```diff
@@ -122,7 +134,7 @@ func Exec(cmdName string, cmdArgs []string, timeout time.Duration) ([]byte, erro
 	// This time we can simply use Output() to get the result.
 	out, err := cmd.Output()
 	if err != nil {
-		ioutils.Error(fmt.Sprint(err) + ": " + stderr.String())
```

Comment:
looks like we are loosing the actual error message now.

Used to be:

```
$ ./riff-cli/riff list
Listing function resources in namespace default

exit status 1: the server doesn't have a resource type "functions"

Error: exit status 1
```

Now, the middle error line is missing